.env.example

```diff
@@ -42,6 +42,10 @@ SALESFORCE_URL=https://salesforce.com
 # Salesforce login user
 SALESFORCE_LOGIN=user@nationalleadership.gov.uk
 
+# The URL of the Elastic Search Cluster
+ELASTICSEARCH_URL=http://nlc_cms_elasticsearch:9200
```

Comment:
I can see this env var being created here in this commit, but no use of that in a file, like e.g. `settings.php` — should something else be committed also?